testing/src/App.test.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import render from "react-test-renderer";
+import App from "/home/sprung/hackreactor/Project-Atelier/client/src/components/App.jsx";
+
+
+describe("App Component", function () {
+  it("should have hello world message", function () {
+    console.log(App);
+    console.log(render.create(<App />));
+    // const component = renderer.create(
+    //   <App page="http://www.facebook.com">Facebook</App>,
+    // );
+    // let tree = component.toJSON();
+    // expect(tree).toMatchSnapshot();
+    // let getByText = renderer(<App />);
+    // expect(getByText("Hello World")).toMatchInlineSnapshot(`
+    //   <h1>
+    //     Hello world
+    //   </h1>
+    // `);
```

Comment:
I'm just wondering about the commented out code...I'm assuming it was an attempt at writing an initial test for App component that you didn't want to delete just yet?

Reply:
Or is it that we took out the Hello World message from the App component?